getstarted.rst

```diff
@@ -154,14 +176,22 @@ Again, if you don't have ssh access set up for GitHub then try using the https f
 Running nio
 ~~~~~~~~~~~
 
-This part is simple. With the virtual environment active (which it should already be if you've been following along), run the following command from the root of your project directory (which should also already be ready if you've been following along):
+This part is simple. Navigate to your project directory (which you should still be in if you've been following along), and run the following command. If you've installed n.io to a virtual environment, make sure it's active!
 
 .. code-block:: bash
 
-    cd ~/nio/projects/getting_started
-    run_nio
+    cd ~/nio/projects/<project_name>
+    nio server
 
-It is common to see many INFO log messages. If you see any ERROR messages you may have a problem. But for now lets ignore this one: `NIO [ERROR] [Discover] Failure loading module nioext.components.snmp.agent ImportError:No module named 'pysnmp'`. We won't be using that anyway.
+It is common to see many INFO log messages. If you see any ERROR messages you may have a problem. But for now lets ignore this one: ``NIO [ERROR] [Discover] Failure loading module nioext.components.snmp.agent ImportError:No module named 'pysnmp'``. We won't be using that anyway.
```

Comment:
We've changed the default logging so we no longer get a lot of INFO messages. Go ahead and remove that first sentence.